x-pack/plugins/security_solution/public/management/pages/endpoint_hosts/view/details/endpoint_activity_log.tsx

```diff
@@ -32,9 +57,20 @@ export const EndpointActivityLog = memo(
           <>
             <SearchBar onSearch={onSearch} placeholder={i18.SEARCH_ACTIVITY_LOG} />
             <EuiSpacer size="l" />
-            {endpointActions.data.map((endpointAction) => (
-              <LogEntry key={endpointAction.action_id} endpointAction={endpointAction} />
-            ))}
+            {activityLoading ? (
+              <EuiLoadingContent lines={3} />
+            ) : (
+              activityLog.type === 'LoadedResourceState' &&
+              activityLog.data?.map((logEntry) => (
+                <LogEntry
+                  key={`${logEntry.item.action_id}-${logEntry.item['@timestamp']}`}
```

Comment:
This was needed since `action_id` might be the same for actions and responses.

Reply:
the key needs to be the `_id` of the document

Reply:
done 4f7d18bee78cdc389ab0bf3cb94271c66b4ae25b